File: GameEngine/GameEngine.CSharp/Game/Engine/GameConfiguration.cs
using System;
using System.Runtime.Serialization;

namespace GameEngine.CSharp.Game.Engine
{
    [DataContract]
    public class GameConfiguration
    {
        [DataMember]
        public int numberOfAI { get; set; }

        [DataMember]
        public Guid GameId { get; set; }
    }
}

File: GameEngine/GameEngine.FSharp/GameConfiguration.fs
namespace GameEngine.FSharp

open System
open System.Runtime.Serialization

[<DataContract>]
type GameConfiguration = {
        [<field: DataMember(Name="numberOfAI") >]
        numberOfAI : int;

        [<field: DataMember(Name="GameId") >]
        GameId : Guid;
    }


